edmc_data.py

```diff
@@ -495,3 +495,77 @@
     'viper_mkiv':                   'Viper MkIV',
     'vulture':                      'Vulture',
 }
+
+# Odyssey Suit Names
+edmc_suit_shortnames = {
+    'Flight Suit':            'Flight',     # EN
+    'Artemis Suit':           'Artemis',    # EN
+    'Dominator Suit':         'Dominator',  # EN
+    'Maverick Suit':          'Maverick',   # EN
+
+    'Flug-Anzug':             'Flug',       # DE
+    'Artemis-Anzug':          'Artemis',    # DE
+    'Dominator-Anzug':        'Dominator',  # DE
+    'Maverick-Anzug':         'Maverick',   # DE
+
+    'Traje de vuelo':         'de vuelo',   # ES
+    'Traje Artemis':          'Artemis',    # ES
+    'Traje Dominator':        'Dominator',  # ES
+    'Traje Maverick':         'Maverick',   # ES
+
+    'Combinaison de vol':     'de vol',     # FR
+    'Combinaison Artemis':    'Artemis',    # FR
+    'Combinaison Dominator':  'Dominator',  # FR
+    'Combinaison Maverick':   'Maverick',   # FR
+
+    'Traje voador':           'voador',     # PT-BR
+    'Traje Artemis':          'Artemis',    # PT-BR
+    'Traje Dominator':        'Dominator',  # PT-BR
+    'Traje Maverick':         'Maverick',   # PT-BR
+
+    'Летный комбинезон':      'Летный',     # RU
+    'Комбинезон Artemis':     'Artemis',    # RU
+    'Комбинезон Dominator':   'Dominator',  # RU
+    'Комбинезон Maverick':    'Maverick',   # RU
```

Comment:
it LOOKS to me as if their localisation for this is just the the word suit in the language and the name of the suit otherwise. Which I guess makes sense? but that being the case we could probably drop this for everything but flight?

Reply:
I feel that doing the two stages (symbol to actual game translation, then that to the short form) gives us the greater flexibility.  This way we, and plugins, can use the 'full form' if they want, and also have this single agreed upon short form as well.

I can see an argument for doing the mapping straight from symbol to short form, rather than via intermediary.  In part I'm being paranoid about Frontier changing a translation to a form where this shortening *doesn't* make sense.  We'd be left with the long form translation as inn-game then, rather than effectively forcing an English name.

Reply:
Yeah in that case leaving it is the best idea.